lld/test/ELF/shuffle-padding-rodata.s

```diff
@@ -0,0 +1,36 @@
+# REQUIRES: x86
```

Comment:
bss/data/rodata can be merged into one test.

It's important to test an output section with multiple InputSectionDescription.

By default .rodata .text .data are in different PT_LOAD segments. The `os->ptLoad != curPtLoad` condition isn't covered by tests.